Python/Product/Analysis/Parsing/Ast/MemberExpression.cs

```diff
@@ -76,7 +76,11 @@ public sealed class MemberExpression : Expression {
             res.Append('.');
             if (!this.IsIncompleteNode(ast)) {
                 res.Append(this.GetSecondWhiteSpaceDefaultNull(ast));
-                res.Append(this.GetVerbatimImage(ast) ?? _name);
+                if (format.UseVerbatimImage) {
+                    res.Append(this.GetVerbatimImage(ast) ?? _name);
```

Comment:
Round tripping is really only going to work in a reasonable way if we parse in verbatim mode, and in that case we'll always have the right verbatim image available if it differs from the real name.

Elsewhere where we deal with this we first do:

            string privatePrefix = GetPrivatePrefixClassName(scope);
            var ast = GetAstFromText(exprText, privatePrefix);

Does that work in this case too?